src/site/content/en/blog/optimize-ttfb/index.md

```diff
@@ -0,0 +1,229 @@
+
+Once you have a good HSTS policy in place, you can speed things up on the first visit to an origin [by adding your site to the HSTS preload list](https://developer.mozilla.org/docs/Web/HTTP/Headers/Strict-Transport-Security#preloading_strict_transport_security).
+
+{% Aside 'caution' %}
+Be **very careful** when implementing HSTS, as setting too aggressive a policy without sufficient testing [can break your website](/bbc-hsts/#deploying-hsts).
+{% endAside %}
+
+### Stream markup to the browser
+
+Browsers are optimized to deliver markup efficiently through streaming, meaning that markup is handled in chunks as it arrives from the server. This is crucial where large markup payloads are concerned, as it means the browser can parse that large chunk of markup incrementally, as opposed to waiting for the entire response to arrive before parsing can begin.
+
+Though browsers are great at streaming markup, it's crucial to do all that you can to keep that stream flowing so those initial bits of markup are on their way as soon as possible. If the back end is holding things up, that's a problem. Because back end stacks are numerous, it would be beyond the scope of this guide to cover every single stack and the issues that could arise in each specific one.
+
+React, for example—and other frameworks that can [render markup on demand on the server](/rendering-on-the-web/#server-rendering)—have used a synchronous approach to server-side rendering. However, newer versions of React have implemented [server methods for streaming markup](https://reactjs.org/docs/react-dom-server.html#overview) as it is being rendered. This means you don't have to wait for a React server API method to render the entire response before it's sent.
+
+{% Aside %}
+Not every JavaScript runtime can take advantage of streaming server-side rendering. Newer JavaScript runtimes such as [Deno](https://deno.land/) support this out of the box, but older versions of [Node.js](https://nodejs.org/) may not support it. Check to see if this is the case for you, and see what you can do to upgrade or switch your runtime for better server-side rendering performance.
+{% endAside %}
+
+Another way to ensure markup is streamed to the browser is to rely on [static rendering](/rendering-on-the-web/#static-rendering) which generates HTML files during build time. While this approach isn't suitable for every page on every website—such as those requiring a dynamic response as part of the user experience—it can be beneficial for those pages that don't require markup to be personalized to a specific user.
```

Comment:
Is this really streaming? Word-smithed to make it more like it is:

```suggestion
Another way to ensure markup is streamed to the browser quickly is to rely on [static rendering](/rendering-on-the-web/#static-rendering) which generates HTML files during build time. With the full file available immediately, web servers can start sending the file immediately and the inherit nature of HTTP will result in streaming markup. While this approach isn't suitable for every page on every website—such as those requiring a dynamic response as part of the user experience—it can be beneficial for those pages that don't require markup to be personalized to a specific user.
```